h/streamer/messages.py

```diff
@@ -107,8 +107,12 @@ def handle_annotation_event(message, sockets, request, session):
         log.warning("received annotation event for missing annotation: %s", id_)
         return
 
+    # Find equivalent URIs. This matches how query URIs are expanded by the
+    # annotation search API. Note that normalization is handled by `SocketFilter.matching`.
+    equivalent_uris = storage.expand_uri(session, annotation.target_uri)
+
     # Find connected clients which are interested in this annotation.
-    matching_sockets = SocketFilter.matching(sockets, annotation)
+    matching_sockets = SocketFilter.matching(sockets, annotation, equivalent_uris)
```

Comment:
I don't think `equivalent_uris` is used out here, could this be private to the method? Presumably this is because of the session being here?